sources/packages/backend/libs/sims-db/src/entities/student-pd-status.type.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Permanent disability status of student.
```

Comment:
Everything is SIMS used to be named using PD as permanent disability which is no longer true with the inclusion of PPD.
I would say that we need to reinforce it for the new things and try to adjust the current ones as possible.
As a suggestion, I would remove the "Permanent" from the comment and name the below enum as `DisabilityStatus`.